src/main/java/Test2.java

```diff
@@ -0,0 +1,36 @@
+
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/Test2")
```

Comment:
It`s better to use specail names. For example: 
`@WebServlet("/request/type")`
You can use a better name  than mine.